Client/mods/deathmatch/logic/CClientRadarMarkerManager.cpp

```diff
@@ -81,14 +81,16 @@ void CClientRadarMarkerManager::RemoveFromList(CClientRadarMarker* pMarker)
 
 void CClientRadarMarkerManager::SetDimension(unsigned short usDimension)
 {
-    m_usDimension = usDimension;
-    m_bOrderOnPulse = true;
+    if (m_usDimension != usDimension)
+        m_usDimension = usDimension;
+        m_bOrderOnPulse = true;
```

Comment:
I think there should be `{}`

Reply:
Probably doesn't have to be

Reply:
> Probably doesn't have to be

`m_bOrderOnPulse = true;` will be executed regardless of the condition.
Do it in this way:
> if (m_usDimension != usDimension)
> {
>         m_usDimension = usDimension;
>         m_bOrderOnPulse = true;
> }

Reply:
Really, thanks.